packages/macro/src/macroJs.ts

```diff
@@ -52,7 +52,7 @@ export type MacroJsOpts = {
   stripNonEssentialProps: boolean
 }
 
-export default class MacroJs {
+export class MacroJs {
```

Comment:
not related to whitespacing, simply eliminating default import/exports since they are harmful in cjs/esm interops